design/vela-core/workflow_policy.md

```diff
@@ -147,13 +147,13 @@ Each workflow step has the following interactions with the app controller:
     - type: workflow-finish
       status: 'True'
       reason: 'Succeeded'
-    - ...
+      message: '{"generation":1}'
```

Comment:
```suggestion
      message: '{"observedGeneration":1}'
```

Reply:
> why don't we use yaml format? 

This is json marshalled data. Yaml is not easy to put in string.

>  json is hard to read and decode mixed with yaml

This is not for human to read and write. This part should be automatically generated by controller.